OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -526,10 +526,16 @@ static class DMOpcodeHandlers {
         }
 
         public static ProcStatus? BitOr(DMProcState state) {
-            int second = state.PopDreamValue().GetValueAsInteger();
-            int first = state.PopDreamValue().GetValueAsInteger();
+            DreamValue value = state.PopDreamValue();
+            if (value.TryGetValueAsInteger(out int secondInt)) {
+                int first = state.PopDreamValue().GetValueAsInteger();
+
+                state.Push(new DreamValue(first | secondInt));
+            } else if (value.TryGetValueAsDreamList(out DreamList secondList)) {
+                DreamList first = state.PopDreamValue().GetValueAsDreamList();
```

Comment:
What if someone mixes an integer and a list?

Reply:
:sadcat: